acceptance-tests/features/add_reference_to_case.feature

```diff
@@ -0,0 +1,44 @@
+@add_ref_to_case
+
+Feature: Add Reference to Case
+  As a conveyancer who has many cases to track I need to be able to mark a case
+  with an internal reference so that I can identify exactly which case on my
+  list matches my own internal case list
+
+Scenario Outline: Conveyancer Enters Reference
+
+  - If a case reference is provided then it will be displayed on the case list
+
+  Given I view the case list
+  And I click on the "Create a new case" link
+  When I enter the case reference <reference>
+  And I click on the "Create case" button
+  Then the "Case list" page is displayed
+  And the reference <reference> is displayed in the case list
+
+  Examples:
+    | reference                                       |
+    | "A"                                             |
+    | "XYZ123"                                        |
+    | "13 High Street, Ernesettle, Plymouth, PL3 8JN" |
```

Comment:
Just checking that this isn't actually your address @nigelpain as this code could be made public at some stage. Just an FYI